### ISSUE TITLE
convert to tf2 and tensorflow-probability

### ISSUE THREAD

**Issue by kaczmarj** [neuronets/kwyk#15]
the kwyk models should be converted to tensorflow 2 and tensorflow-probability. this will make transfer learning / hacking easier for more users.

### instructions to extract the pre-trained weights from saved models

this works with tensorflow 1.13.1 and uses the saved models in https://github.com/patrick-mcclure/nobrainer.

the spike and slab model uses variables `p` in concrete dropout, whereas the other two models do not.

```python
import numpy as np
import tensorflow as tf

# Load saved model.
export_dir = 'saved_models/all_50_bvwn_multi_prior/1556816070/'
predictor = tf.contrib.predictor.from_saved_model(export_dir)
# Get pre-trained weights.
ps = predictor.session.run(predictor.graph.get_collection('ps'))
ms = predictor.session.run(predictor.graph.get_collection('ms'))
sigmas = predictor.session.run(predictor.graph.get_collection('sigmas'))
# Save to file.
np.savez_compressed('spike-slab.npz', ms=ms, sigmas=sigmas, ps=ps)
```

related to https://github.com/neuronets/nobrainer-models/issues/3

**Comment by kaczmarj** [comment]
closed by https://github.com/neuronets/nobrainer/pull/80
